Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM python:3.8-buster
```

Comment:
We are not working with Python 3.8 yet because there is an issue with aiohttp

Reply:
Are you talking about this issue aio-libs/aiohttp#4549? I didn't know since I never experience this problem. Maybe we should add a failing test to update the python version when it stops to fail.

Reply:
Done.